### PR TITLE
Upgraded grafter to 2.1.0

### PR DESCRIPTION
**Pull request by etorreborre**
Review @jcranky

### CODE REVIEW

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -8,9 +8,11 @@ val circeVersion = "0.7.1"
 val http4sVersion = "$http4s_version$"
 val grafterVersion = "$grafter_version$"
 val logbackVersion = "1.2.3"
+val scalametaVersion = "1.8.0"
 
 libraryDependencies ++= Seq(
   "org.zalando"    %% "grafter"             % grafterVersion,
+  "org.scalmeta"   %% "contrib"             % scalametaVersion,
```
Comment (jcranky) [inline src/main/g8/build.sbt]: `scalmeta`?

Comment (etorreborre) [reply src/main/g8/build.sbt]: fixed.

**Comment by jcranky** [comment]
@etorreborre I got a `org.zalando#grafter_2.12;2.0.1: not found` when trying this locally, is 2.0.1 already published?

**Comment by etorreborre** [comment]
Not yet, I want to make sure it works on tinbox first. But you can checkout the branch and publish locally.

**Comment by jcranky** [comment]
@etorreborre I see, I'll wait that to merge then. Thanks :D

**Comment by jcranky** [comment]
This would now have to be updated to 2.1.x, right ?

**Comment by etorreborre** [comment]
Yes, I'll do it.

**Comment by etorreborre** [comment]
@jcranky done.

**Comment by jcranky** [comment]
Thanks. The generated project is not compiling yet though:

```
[info] Compiling 10 Scala sources to /tmp/grafter-http4s-example/target/scala-2.12/classes...
[error] [1] src/main/scala/com/example/grafterhttp4sexample/main/Application.scala:13:
[error]     type mismatch;
[error]      found   : com.example.grafterhttp4sexample.main.ApplicationConfig
[error]      required: cats.data.Reader[?,com.example.grafterhttp4sexample.HttpServer]
[error]         (which expands to)  cats.data.Kleisli[cats.Id,?,com.example.grafterhttp4sexample.HttpServer]
[error]         val components = ApplicationComponents.reader(config).configure(config)
[error]                                                       ^
[error] [2] src/main/scala/com/example/grafterhttp4sexample/main/Visualize.scala:7:
[error]     type mismatch;
[error]      found   : com.example.grafterhttp4sexample.main.ApplicationConfig
[error]      required: cats.data.Reader[?,com.example.grafterhttp4sexample.HttpServer]
[error]         (which expands to)  cats.data.Kleisli[cats.Id,?,com.example.grafterhttp4sexample.HttpServer]
[error]       val components = ApplicationComponents.reader(config).configure(config)
[error]                                                     ^
[error] src/main/scala/com/example/grafterhttp4sexample/main/Visualize.scala: 7 [2]
[error] src/main/scala/com/example/grafterhttp4sexample/main/Application.scala: 13 [1]
[error] (compile:compileIncremental) Compilation failed

```

**Comment by etorreborre** [comment]
This should be fixed now.

**Comment by jcranky** [comment]
awesome, thanks! :)

**Pull request merged**